react/features/participants-pane/components/web/VisitorsList.tsx

```diff
@@ -67,6 +67,9 @@ const useStyles = makeStyles()(theme => {
 export default function VisitorsList() {
     const requests = useSelector(getPromotionRequests);
     const visitorsCount = useSelector((state: IReduxState) => state['features/visitors'].count || 0);
+    const visitorsInQueueCount = useSelector((state: IReduxState) => state['features/visitors'].inQueueCount || 0);
```

Comment:
Maybe we should create a function for this seletor since it is used in 2+ places?

Reply:
Done.